ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__nfd.yaml

```diff
@@ -38,6 +38,25 @@ tests:
           cpu: 2000m
           memory: 2Gi
     workflow: generic-claim
+- as: nightly-e2e-master
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h30m0s
+    version: "4.10"
+  cron: 0 23 * * *
+  steps:
+    test:
+    - as: nightly
+      commands: run prow nfd-operator test_master_branch
+      from: ci-artifacts
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 2Gi
+    workflow: ipi-aws
```

Comment:
```
    workflow: generic-claim
```

Reply:
when you take the cluster from a pool (`cluster_claim`), you need `workflow: generic-claim`

Reply:
while `ipi-aws` is when you want it built

Reply:
... but in Periodics, you can't use the pool IIRC